cinema/views.py

```diff
@@ -34,7 +40,36 @@ class MovieViewSet(viewsets.ModelViewSet):
     queryset = Movie.objects.all()
     serializer_class = MovieSerializer
 
-    def get_serializer_class(self):
+    @staticmethod
+    def _params_to_list_ints(qs: str) -> list[int]:
+        return [int(int_id) for int_id in qs.split(",")]
+
+    def get_queryset(self) -> QuerySet:
+        queryset = self.queryset
+        genres = self.request.query_params.get("genres")
+        actors = self.request.query_params.get("actors")
+        title = self.request.query_params.get("title")
+
+        if genres:
+            genres = self._params_to_list_ints(genres)
+            self.queryset = self.queryset.filter(genres__id__in=genres)
+
+            return self.queryset.distinct()
```

Comment:
Redundant return. What if I want to filtrer by genres and actors simultaneously?